src/ray/core_worker/shutdown_coordinator.h

```diff
@@ -248,6 +214,23 @@ class ShutdownCoordinator {
   std::string GetReasonString() const;
 
  private:
+  /// Legacy method for compatibility - delegates to RequestShutdown
```

Comment:
what does "compatibility" mean here? this is a private method that was introduced only recently